src/NzbDrone.Core/Update/UpdateCheckService.cs

```diff
@@ -30,13 +31,13 @@ public class CheckUpdateService : ICheckUpdateService
 
         public UpdatePackage AvailableUpdate()
         {
+            var latestAvailable = _updatePackageProvider.GetLatestUpdate(_configFileProvider.Branch, BuildInfo.Version);
+
             if (OsInfo.IsNotWindows && !_configFileProvider.UpdateAutomatically)
```

Comment:
Okay good, I wasn't going crazy, which means we also might have a lot more users... on old versions.